src/mudlet.pro

```diff
@@ -175,6 +175,18 @@ linux|macx|win32 {
 # else we are on another platform which the updater code will not support so
 # don't include it either
 
+
+######################### 3D mapper toggle #########,#############
+# To remove the 3D mapper, set the environment WITH_3DMAPPER variable to "NO"
+# ie: export WITH_3DMAPPER="NO" qmake
+#
+linux|macx|win32 {
```

Comment:
Please add the `freebsd` (check elsewhere in the same file to confirm that that *is* the word) scope here as the 3D mapper is supported there.

Reply:
Can you confirm that's the word?

Reply:
I´ll try and let you know in the next day or so.

Reply:
... though I am sure it appears elsewhere in the file.

Reply:
:bulb: We do NOT want to include a platform scope test here - the Raspberry Pi will *probably* be running a Linux OS (though others are available - I am aware of a RaspBSD {FreeBSD derivative} build and of course there is also Windows IOT - {non-GUI?}, RISC OS, Plan 9, etc.)

Bottom line is that checking for the OS is not helpful here (or in the CMake equivalent) IMHO.

Reply:
I think it is, for the same reason as the updater code does. If we run into it being an issue, we can change the check.